méli-mélo/2023-09-menu/campaign-menu.css

```diff
@@ -1,7 +1,7 @@
 /*** Mega Menu ***/
 /* add background and text color for menu */
 #wb-sm.campaign-menu {
-    background: #003c77;
+    background: #000;
```

Comment:
After discussion,

Let keep the blue as the default color and you can add a feature which are going to support a generic "bg-darker" variant. We will need to have an example of both style.

Reply:
Fixed: GC theme blue is the default colour, and black is available as a feature/option.